### PR TITLE
Webpack conversion

### PR DESCRIPTION
**Pull request by richardzeng1**
Converted TA-Toy to Vue CLI

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,38 +1,13 @@
...
-        </div>
-        <script src="js/component-courses.js"></script>
-        <script src="js/component-tas.js"></script>
-        <script src="js/component-sections.js"></script>
-        <script src="js/component-pageNotFound.js"></script>
-        <script src="js/app.js"></script>
-    </body>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```
Comment (siefkenj) [inline index.html]: 4 space indentation

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "ta-toy",
```
Comment (siefkenj) [inline package.json]: 4 spaces

File: src/App.vue [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<template>
+  <div id="app">
```
Comment (siefkenj) [inline src/App.vue]: spaces

File: src/App.vue [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    <div id="app">
+        <div id="page">
+            <navbar>
+                <h1>Tutorial Feedback</h1>
+            </navbar>
+            <div id="container">
+                <router-view></router-view>
+            </div>
+            <div id="footer">
+                <p>University of Toronto
```
Comment (siefkenj) [inline src/App.vue]: Should end `</p>` on the same line.

File: src/App.vue [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+                </p>
+            </div>
+        </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
```
Comment (siefkenj) [inline src/App.vue]: I don't think this should be here.

File: src/App.vue [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+            msg: "Welcome to Your Vue.js App"
+        };
+    }
+};
+</script>
+
+<style>
+#app {
+    font-family: "Avenir", Helvetica, Arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
```
Comment (siefkenj) [inline src/App.vue]: Don't use browser prefixes in css

File: src/App.vue [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    name: "app",
+    data() {
+        return {
+            msg: "Welcome to Your Vue.js App"
+        };
+    }
+};
+</script>
+
+<style>
```
Comment (siefkenj) [inline src/App.vue]: I don't like having the style inline for the main app. Leave it as `app.css`. Only use inline styles for specific components.

File: src/components/component-courses.vue [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+            })
+            .catch(error => {
+                this.error = true;
+                return Promie.reject();
+            });
+    }
+};
+</script>
+
+<style>
```
Comment (siefkenj) [inline src/components/component-courses.vue]: Is any of this style needed?

File: src/components/component-sections.vue [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+            .catch(err => {
+                this.error = true;
+                this.loading = false;
+                return Promise.reject();
+            });
+    }
+};
+</script>
+
+<style>
```
Comment (siefkenj) [inline src/components/component-sections.vue]: Same comment

File: src/components/component-tas.vue [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+            .catch(err => {
+                this.error = true;
+                this.loading = false;
+                return Promise.reject();
+            });
+    }
+};
+</script>
+
+<style>
```
Comment (siefkenj) [inline src/components/component-tas.vue]: Again

File: webpack.config.js [hunk 11]
```diff
@@ -0,0 +1,85 @@
+var path = require('path')
+var webpack = require('webpack')
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+var HTMLWebpackPluginConfig = new HtmlWebpackPlugin({
+  template: __dirname + '/index.html',
```
Comment (siefkenj) [inline webpack.config.js]: prettier, 4 spaces.

File: src/App.vue (hunk 5 above)
Comment (richardzeng1) [reply src/App.vue]: Can you elaborate. All the examples I've seen online use export default

File: LICENSE [hunk 12]
```diff
@@ -1,674 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```
Comment (siefkenj) [inline LICENSE]: Why did you remove the license?
